docker-compose.yml

```diff
@@ -666,7 +666,7 @@ services:
     hostname: exc1
     <<: *x-logging
     stop_signal: SIGINT
-    image: openruntimes/executor:0.1.6
+    image: myexc
```

Comment:
Why did we change this ?

Reply:
We need new version here. I changed for testing purposes. Once OPR is reelased, we can revert and increase version